pyrobosim/pyrobosim/core/room.py

```diff
@@ -30,7 +33,8 @@ def __init__(
         :type name: str, optional
         :param footprint: Point list or Shapely polygon describing the room 2D footprint (required).
         :type footprint: :class:`shapely.geometry.Polygon`/list[:class:`pyrobosim.utils.pose.Pose`]
-        :param color: Visualization color as an (R, G, B) tuple in the range (0.0, 1.0)
+        :param color: Visualization color as an (R, G, B) tuple in the range (0.0, 1.0),
+                      a string (e.g., "red"), or a hexadecimal (e.g., "#FF0000").
         :type color: (float, float, float), optional
```

Comment:
make sure you modify the `type` as well to include the string alternative

Reply:
Done.